cli/context/context.go

```diff
@@ -173,3 +174,14 @@ type FailoverCtx struct {
 	ParamsJSON         string
 	ProviderParamsJSON string
 }
+
+type BenchCtx struct {
+	URL                  string
+	User                 string
+	Password             string
+	Connections          int
+	SimultaneousRequests int
```

Comment:
`// SimultaneousRequests describes the number of parallel requests from one connection.`